packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -274,10 +274,14 @@ class BottomSheet extends Component {
 
 	onHeaderLayout( { nativeEvent } ) {
 		const { height } = nativeEvent.layout;
+		// The layout animation should only be triggered if the header
+		// height has changed after being mounted.
+		if ( this.headerHeight !== 0 && height !== this.headerHeight ) {
+			this.performRegularLayoutAnimation( {
+				useLastLayoutAnimation: true,
+			} );
+		}
```

Comment:
`onHeaderLayout` is executed when the bottom-sheet component is mounted so if the component doesn't make any change in the layout, any other change made after it was closed would be animated. This doesn't particularly introduce a bug but I found out not consistent that some interactions were animated and others not.